rmf_sandbox/src/camera.rs

```diff
@@ -0,0 +1,35 @@
+use bevy::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Component, Clone, Default)]
+pub struct Camera {
```

Comment:
In Bevy 0.8 there's a new component called `Camera` in `bevy::prelude::*` so I think that's going to cause some naming conflicts with this struct.

Maybe we can name this struct `PhysicalCamera` instead?

Reply:
It looks like [Bevy 0.7 already has `Camera`](https://docs.rs/bevy/0.7.0/bevy/prelude/struct.Camera.html), how is it working now?

Since we want to differentiate between the camera that controls what is visible on the UI and the cameras inserted into the scene, I think `PhysicalCamera` or `CameraEntity` make sense